packages/desktop-client/src/components/reports/reports/Spending.tsx

```diff
@@ -139,48 +139,115 @@ export function Spending() {
       <View
         style={{
           flexDirection: 'row',
+          alignItems: 'center',
+          padding: 20,
+          paddingBottom: conditions.length > 0 ? 0 : 10,
           flexShrink: 0,
         }}
       >
         <View
           style={{
-            padding: 20,
-            flexShrink: 0,
+            alignItems: 'center',
+            flexDirection: 'row',
+            marginRight: 5,
           }}
         >
-          {conditions && (
-            <View style={{ flexDirection: 'row' }}>
-              <FilterButton
-                onApply={onApplyFilter}
-                compact={false}
-                hover={false}
-                exclude={['date']}
-              />
-              <Tooltip
-                placement="bottom start"
-                content={<Text>Save compare and filter options</Text>}
-                style={{
-                  ...styles.tooltip,
-                  lineHeight: 1.5,
-                  padding: '6px 10px',
-                  marginLeft: 10,
-                }}
-              >
-                <Button
-                  type="primary"
-                  style={{
-                    marginLeft: 10,
-                  }}
-                  onClick={saveFilter}
-                  disabled={filterSaved ? true : false}
-                >
-                  {filterSaved ? 'Saved' : 'Save'}
-                </Button>
-              </Tooltip>
-              <View style={{ flex: 1 }} />
-            </View>
+          <strong
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏maybe we shouldn't make the label bold. I just compared it with the other reports and we don't use bold anywhere else. WDYT?

Reply:
Sounds good!